CHANGELOG.md

```diff
@@ -5,6 +5,15 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v1.2.1 (TBD)
```

Comment:
Isn't it a mistake? Latest is 1.1.6

Reply:
(check a few lines below)

I believe @Konstantinov-Innokentii merged `dev` -> `main` earlier today, but hasn't deployed/released these changes (which would be `v1.2.0`). I suspect this commit won't make it into `v1.2.0`?

Reply:
or I can include it under the changes for `v1.2.0` and we'll just need to remember to do another `dev` -> `main` merge before doing a full release

Reply:
decided to just include it under `v1.2.0`, I'll be sure to do another `dev` -> `main` merge before releasing on Monday morning